### ISSUE TITLE
Show the same series on all dashboard cards on hover

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#48008]
**Is your feature request related to a problem? Please describe.**
Some tools have this and a customer requested if we could show the same series on all dashcards when you hover over those. E.g. let's say that I build 4 cards for a dashboard that has the "Gizmo" series or category. If Gizmo is on multiple cards, we should focus on those in other cards when you hover over any

**Describe the solution you'd like**
Above

**Describe alternatives you've considered**
None

**How important is this feature to you?**
Requested by a customer since they had that in a previous tool

**Additional context**
NA

**Comment by brunobergher** [comment]
I think this is pretty much the same as #48041, so closing as duplicate. Let me know otherwise.

**Comment by paoliniluis** [comment]
Nope, seem like that one is for the same chart and I was looking for the feature on all the charts in the dashboard that have the same series

**Comment by brunobergher** [comment]
That's what that is: "Synchronize the hover state of **multiple cartesian charts** in the same dashboard"
